client/app/views/index.js

```diff
@@ -24,11 +24,17 @@ function fetchPipelineIds() {
 }
 
 function appendPipeline(pipeline) {
-  dom('#content-container').append(renderPipeline(pipeline));
+  var pipelineId = pipeline.name.toLowerCase().replace(' ', '-');
```

Comment:
@viniciusgama With your last change I think this is not necessary anymore right? Also the addtion of name attribute in the tests. Does it make sense?